mod.ts

```diff
@@ -42,19 +54,25 @@ export class Coffee {
 
   configs: Configs = {};
 
-  load(options: LoadOptions = this.defaultOptions): void {
-    const rawConfigs = this.runtimeAPI.readFile(
-      options.configPath + "/default.json",
+  load(opts: LoadOptions = {}): void {
+    this.defaultOptions = deepExtend(this.defaultOptions, opts);
```

Comment:
do we need `deepExtend` here? and the `defaultOptions` as a name do not make sense here while manipulating.

Reply:
because we should merge both configs into one, not just using one of them

Reply:
since neither of these is nested why not just `Object.assign`? it's a more JSy way and less complex.
and about the name, if we change the default options, it's not `default` anymore.
it can be something like `startupOptions`.

Reply:
`Object.assign` is not a deep merger/extender, if we have nested config structure, It doesn't work anymore.
About name, I Agree with U, better to be `options`